pandas/tests/groupby/test_value_counts.py

```diff
@@ -52,29 +52,28 @@ def seed_df(seed_nans, n, m):
 
 @pytest.mark.slow
 @pytest.mark.parametrize("df, keys, bins, n, m", binned, ids=ids)
-def test_series_groupby_value_counts(df, keys, bins, n, m):
+@pytest.mark.parametrize(
+    "isort, normalize, sort, ascending, dropna", list(product((False, True), repeat=5))
```

Comment:
Not really a fan of using `itertools` to generate parametrization. Could you instead write this as:

```python
@pytest.mark.parametrize("isort", [True, False])
@pytest.mark.parametrize("normalize", [True, False])
...
```

Equivalent at the end of the day but I think clearer and match the rest of the codebase